tests/integration/test_api.py

```diff
@@ -420,29 +420,31 @@ def test_prediction_collection_no_filter(client):
 
 
 def test_get_predictions():
-    prediction1 = PredictionFactory(barcode="123")
-    prediction2 = PredictionFactory(type="category")
-    prediction3 = PredictionFactory(value_tag="en:seeds")
-    prediction4 = PredictionFactory(type="brand")
+    PredictionFactory(barcode="123", keep_types="category", value_tag="en:seeds")
+    PredictionFactory(type="category")
+    PredictionFactory(value_tag="en:seeds")
+    PredictionFactory(type="brand")
 
     actual_prediction1 = get_predictions(barcode="123")
     actual_items1 = [item.to_dict() for item in actual_prediction1]
     assert actual_items1[0]["barcode"] == "123"
     assert actual_items1[0]["type"] == "category"
     assert actual_items1[0]["value_tag"] == "en:seeds"
+    assert len(actual_items1) == 1
 
-    actual_prediction2 = get_predictions(keep_types="category")
+    actual_prediction2 = get_predictions(keep_types="brand")
```

Comment:
I cannot pass the `keep_types` parameter to `get_predictions`. It throws an error in the query.

Reply:
```suggestion
    actual_prediction2 = get_predictions(keep_types=["brand"])
```